integration_tests/test_program.py

```diff
@@ -21,3 +24,21 @@ def test_codemods_include_exclude_conflict(self):
             check=False,
         )
         assert completed_process.returncode == 3
+
+    def test_load_sast_only_by_flag(self, tmp_path):
+        tmp_file_path = tmp_path / "sonar.json"
+        tmp_file_path.touch()
+        completed_process = subprocess.run(
+            [
+                "codemodder",
+                "tests/samples/",
+                "--sonar-issues-json",
+                f"{tmp_file_path}",
+                "--dry-run",
+            ],
+            check=False,
+            capture_output=True,
+            text=True,
+        )
+        assert completed_process.returncode == 0
+        assert RemoveAssertionInPytestRaises.id not in completed_process.stdout
```

Comment:
can you explain the goal of this assertion?

Reply:
The goal is to make sure no non-sast codemods are loaded while the `--sarif` or `--sonar-...` flags are passed. Couldn't think of a better way other than looking for the codemod ids printed to stdout. `RemoveAssertionInPytestRaises` just happens to the one I've used to check it.